public/app/plugins/datasource/cloudwatch/components/ConfigEditor/SecureSocksProxySettingsNewStyling.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Added this file because the SecureSocksProxySettings that Cloudwatch uses comes from grafana/ui. We shouldn't change anything there because other datasources use it and changing the design if the `awsDatasourcesNewFormStyling` feature toggle is enabled would change other datasources' config editors. 
The old forms in Cloudwatch still use the component from grafana/ui